src/ListGroup.js

```diff
@@ -54,6 +74,7 @@ const ListGroup = React.forwardRef((props, ref) => {
         className,
         bsPrefix,
         variant && `${bsPrefix}-${variant}`,
+        layout && `${horizontalPrefix}-${layout}`,
```

Comment:
why is `horizontalPrefix` different from just `bsPrefix` above? i think they should be the same?

Reply:
Ah not exactly, in this case it's because horizontalPrefix just prepends `list-group-` to each class, so the resulting classes are `list-group-horizontal*`. Should I rename it/make it more clear?

Reply:
But `horizontalPrefix` is just `'list-group'`. It's the same as `bsPrefix` here. The `horizontal-` part is in `layout`.

Reply:
Ah, I looked it over again and totally my fault - I misunderstood what you meant. I'll update the PR now!